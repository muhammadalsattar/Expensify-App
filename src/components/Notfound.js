import React from "react"
import { Link } from "react-router-dom";
const Notfound = () => (
    <h1>NOT FOUND - <Link to="/">BACK</Link></h1>
)

export default Notfound;