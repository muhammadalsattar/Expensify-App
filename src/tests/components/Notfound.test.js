import React from 'react'
import {shallow} from 'enzyme'
import Notfound from '../../components/Notfound'

test('Should render Not found page', () => {
    const wrapper = shallow(<Notfound/>)
    expect(wrapper).toMatchSnapshot()
})