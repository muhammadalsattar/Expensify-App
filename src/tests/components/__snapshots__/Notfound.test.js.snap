// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Not found page 1`] = `
<h1>
  NOT FOUND - 
  <Link
    to="/"
  >
    BACK
  </Link>
</h1>
`;
